tests/python/gpu/test_operator_gpu.py

```diff
@@ -127,15 +130,15 @@ def check_ifft(shape):
             init_complex.real[:,i] = init[0][:,2*i]
             init_complex.imag[:,i] = init[0][:,2*i+1]
         a = np.fft.ifft(init_complex, n=None, axis=-1, norm=None)
-        assert_almost_equal(a.real, out1[0]/shape_old[1],rtol=1e-3, atol=1e-5)
+        assert_almost_equal(a.real, out1[0]/shape_old[1],rtol=1e-3, atol=1e-12)
```

Comment:
Why are you also making changes to other tests? I think `test_ifft` and `test_fft` are fixed by some other PR already.